spec/tracking_spec.js

```diff
@@ -0,0 +1,92 @@
+describe('App Tracking', function() {
+  var tracking = require('tracking'),
+      sandbox  = sinon.sandbox.create(),
+      client = { invoke: function() {} },
+      eventListener;
+
+  beforeEach(function() {
+    sandbox.stub(client, 'invoke');
+    eventListener = sinon.spy(window, 'addEventListener');
+    tracking.setup(client);
+  });
+
+  afterEach(function() {
+    window.addEventListener.restore();
+    sandbox.restore();
+  });
+
+  function findEvent(event) {
+    for(var i = 0; i < eventListener.args.length; i++) {
+      if(eventListener.args[i][0] === event) {
+        return eventListener.args[i];
+      }
+    }
+  }
+
+  describe('for click events', function() {
+    it('attaches the click event listener', function() {
+      var clickEvent = findEvent('click');
+
+      expect(clickEvent).not.to.be.undefined;
```

Comment:
Does this expectation short circuit the whole it block? If not then if this fails it will cause exceptions on the next two lines.